src/constants.rs

```diff
@@ -86,3 +87,49 @@ impl Display for ResponseType {
         write!(f, "{self:?}")
     }
 }
+
+#[derive(Debug, Deserialize_repr, Serialize_repr, Copy, Clone)]
+#[repr(u16)]
+pub enum SendRemindersTypes {
+    Disable = 0,
+    Enable = 1,
+}
+
+impl Display for SendRemindersTypes {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(f, "{:?}", *self as u16)
+    }
+}
+
+#[derive(Debug, Serialize)]
+pub struct InvoiceItem<'i> {
+    pub amount: f64,
+    pub item_name: &'i str,
+}
+
+impl<'i> Display for InvoiceItem<'i> {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(f, "{self:?}")
+    }
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Invoice<'i> {
+    pub amount: f64,
+    pub account_reference: &'i str,
+    pub billed_full_name: &'i str,
+    pub billed_period: &'i str,
+    pub billed_phone_number: &'i str,
+    pub due_date: DateTime<Utc>,
+    pub external_reference: &'i str,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub invoice_items: Option<Vec<InvoiceItem<'i>>>,
+    pub invoice_name: &'i str,
+}
+
+impl<'i> Display for Invoice<'i> {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(f, "{self:?}")
```

Comment:
See how best this implementation can be improved.

Remember, `Display` defines how an instance of a type should be formatted for end-user consumption while `Debug`defines how an instance of a type should be formatted for debugging purposes.

Reply:
Opted to print out some of the fields, similar to the suggestion made for `InvoiceItems`

Reply:
Cool, that works 👍🏽